lib/dalli/protocol/meta.rb

```diff
@@ -125,8 +126,21 @@ def cas(key)
 
       # Storage Commands
       def set(key, value, ttl, cas, options)
-        write_storage_req(:set, key, value, ttl, cas, options)
-        response_processor.meta_set_with_cas unless quiet?
+        simple_set = options&.key?(:simple_set) ? options.delete(:simple_set) : false
```

Comment:
I want to rethink and rewrite how we use the `RequestFormatter` by building up strings first and then passing them to the socket we often end up making new strings and allocating more objects. We have found when working directly with the IO object we can avoid much of this overhead.... but for now we want to just test it out and capture some performance data.